tsconfig.json

```diff
@@ -17,7 +17,11 @@
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "strict": true,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "paths": {
+      "@pfseed/*": ["src/App/*"],
```

Comment:
might be good to use more generic terms for the pathing since these will be used by others as a baseline - maybe src or app for the first and assets for the latter?